### PR TITLE
added missing test 

### PR DESCRIPTION
**Pull request by j042**
helper classes didn't have tests. This one adds one for `data.py`. Related to #202

### CODE REVIEW

File: tests/test_helpers/test_data.py [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+    # 1. key
+    assert dataholder.get("a") == 1
+    # 2. key and default
+    assert dataholder.get("b", 2) == 2
+    # 3. key and wrong default
+    assert dataholder.get("b", 3) == 2
+    # 4. empty key - always None
+    assert dataholder.get("c") is None
+    # 5. empty key and default
+    assert dataholder.get("c", "123") == "123"
```
Comment (clemens-fricke) [inline tests/test_helpers/test_data.py:158]: Could we have gotten this functionality through inheritance from a build-in class, like default dict?

**Review by clemens-fricke — APPROVED** [review]
Looks good.

File: tests/test_helpers/test_data.py [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+    ta -= 3
+    assert ta.modified
+
+    ta = new_tracked_array()
+    ta *= 1
+    assert ta.modified
+
+    ta = new_tracked_array()
+    ta /= 1.5
+    assert ta.modified
```
[the remark this answers is not in the document]
Comment (clemens-fricke) [reply tests/test_helpers/test_data.py:57]: That is in the new_tracked_array() function. At least he sets it to `False` in the function.

File: tests/test_helpers/test_data.py (hunk 1 above)
Comment (j042) [reply tests/test_helpers/test_data.py:158]: could, yes, I just didn't wanna inherit from the built-in classes

**Pull request merged**